providers/pool.rb

```diff
@@ -120,120 +120,150 @@ def site_identifier
+    configure_application_pool(is_new_set_profile_environment, "processModel.setProfileEnvironment:#{@new_resource.set_profile_environment}")
+    configure_application_pool(is_new_logon_type, "processModel.logonType:#{@new_resource.logon_type}")
+    configure_application_pool(is_new_manual_group_membership, "processModel.manualGroupMembership:#{@new_resource.manual_group_membership}")
+    configure_application_pool(is_new_idle_timeout, "processModel.idleTimeout:#{@new_resource.idle_timeout}")
+    configure_application_pool(is_new_shutdown_time_limit, "processModel.shutdownTimeLimit:#{@new_resource.shutdown_time_limit}")
+    configure_application_pool(is_new_startup_time_limit, "processModel.startupTimeLimit:#{@new_resource.startup_time_limit}")
+    configure_application_pool(is_new_pinging_enabled, "processModel.pingingEnabled:#{@new_resource.pinging_enabled}")
+    configure_application_pool(is_new_ping_interval, "processModel.pingInterval:#{@new_resource.ping_interval}")
+    configure_application_pool(is_new_ping_response_time, "processModel.pingResponseTime:#{@new_resource.ping_response_time}")
+    
+    # recycling items
+    configure_application_pool(is_new_log_event_on_recycle, "recycling.logEventOnRecycle:PrivateMemory,Memory,Schedule,Requests,Time,ConfigChange,OnDemand,IsapiUnhealthy")
+    configure_application_pool(@new_resource.private_mem && is_new_private_memory, "recycling.periodicRestart.privateMemory:#{@new_resource.private_mem}")
+    configure_application_pool(is_new_disallow_rotation_on_config_change, "recycling.disallowRotationOnConfigChange:#{@new_resource.disallow_rotation_on_config_change}")
+    configure_application_pool(is_new_disallow_overlapping_rotation, "recycling.disallowOverlappingRotation:#{@new_resource.disallow_overlapping_rotation}")
+
+    # failure items
+    configure_application_pool(is_new_load_balancer_capabilities, "failure.loadBalancerCapabilities:#{@new_resource.load_balancer_capabilities}")
+    configure_application_pool(is_new_orphan_worker_process, "failure.orphanWorkerProcess:#{@new_resource.orphan_worker_process}")
+    configure_application_pool(@new_resource.orphan_action_exe && is_new_orphan_action_exe, "failure.orphanActionExe:#{@new_resource.orphan_action_exe}")
+    configure_application_pool(@new_resource.orphan_action_params && is_new_orphan_action_params, "failure.orphanActionParams:#{@new_resource.orphan_action_params}")
+    configure_application_pool(is_new_rapid_fail_protection, "failure.rapidFailProtection:#{@new_resource.rapid_fail_protection}")
+    configure_application_pool(is_new_rapid_fail_protection_interval, "failure.rapidFailProtectionInterval:#{@new_resource.rapid_fail_protection_interval}")
+    configure_application_pool(is_new_rapid_fail_protection_max, "failure.rapidFailProtectionMaxCrashes:#{@new_resource.rapid_fail_protection_max_crashes}")
```

Comment:
thanks for catching all these, coding it all in 1 hour and then not testing it definitely makes it hard. i really appreciate it.